.github/workflows/ci-docs.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       # We only use a non-zero build # when making multiple manual builds in one day.
       run: |
         if test -z "${{ github.event.inputs.version }}"; then
-          export VERSION_NUMBER=11.3.$((`git log -n 1 --format=%ct` / (60*60*24)))
+          export VERSION_NUMBER=11.90.$((`git log -n 1 --format=%ct` / (60*60*24)))
```

Comment:
The PR==commit description needs to mention this rather important change: bumping the version to 11.90 due to breaking compatibility.

Reply:
Done.